feature/bgp/bgp.go

```diff
@@ -46,6 +46,12 @@ func (b *BGP) WithAS(as uint32) *BGP {
 	return b
 }
 
+// WithAfiSafiGlobal sets the AFI-SAFI type for BGP global.
+func (b *BGP) WithAfiSafiGlobal(name oc.E_BgpTypes_AFI_SAFI_TYPE) *BGP {
+	b.oc.GetOrCreateBgp().GetOrCreateGlobal().GetOrCreateAfiSafi(name).Enabled = ygot.Bool(true)
+	return b
+}
```

Comment:
Can we update bgp_test.go to include this test case?

Reply:
Added a testcase in bgp_test.go as well! Done!